Dockerfile

```diff
@@ -2,8 +2,6 @@ FROM nvidia/cuda:8.0-devel-ubuntu16.04
 MAINTAINER support@arrayfire.com
 
 RUN apt-get update && apt-get install -y software-properties-common && \
-    add-apt-repository ppa:george-edison55/cmake-3.x && \
-    apt-get update && \
```

Comment:
It is a good idea to keep `apt-get update` even if we aren't adding a repository.